frontend/src/contexts/AuthContext.tsx

```diff
@@ -30,6 +30,7 @@ type AuthContextType = {
   login: (email: string, password: string) => void;
   logout: () => void;
   user: User | null;
+  setUser: (data: User) => void;
```

Comment:
should it be using this `React.Dispatch<React.SetStateAction<User | null>>` ? for the type

Reply:
oh ok